### ISSUE TITLE
Boktai - The Sun Is In Your Hand: Unknown RTC Command

### ISSUE THREAD

**Issue by InfiniteParallel** [visualboyadvance-m/visualboyadvance-m#558]
### ROM details

Which games are affected by the issue (please state if GBA or GB game): Boktai: The Sun Is In Your Hand USA, GBA

Which games, if any, are NOT affected by the issue (please state if GBA or GB game): No others that I am aware of so far

Include screenshots of ROM Information (File > ROM Information): 
![ROM Information](https://user-images.githubusercontent.com/56567725/68079065-a3c6c280-fdb9-11e9-823c-287a9d716410.png)

If you are using a BIOS file, please say which one: Happens if I use a BIOS file or not.

### Option details

Video driver (simple, opengl): OpenGL

Sound driver (directsound, xaudio, openal, SDL): DirectSound

### Build details

Operating System (Windows, Mac, Linux (state distribution), etc.): Windows 10, 1903

Version of code (pre-built binary version, or commit ref, or just "master"): 2.1.4 binary from Releases here on Github

### Hardware details: 

[System Summary]

Item	Value	
OS Name	Microsoft Windows 10 Home	
Version	10.0.18362 Build 18362	
Other OS Description 	Not Available	
OS Manufacturer	Microsoft Corporation	
System Name	SYSTEMBETA	
System Manufacturer	HP	
System Model	HP Notebook	
System Type	x64-based PC	
System SKU	X0H86UA#ABA	
Processor	Intel(R) Core(TM) i3-6100U CPU @ 2.30GHz, 2301 Mhz, 2 Core(s), 4 Logical Processor(s)	
BIOS Version/Date	Insyde F.47, 5/27/2019	
SMBIOS Version	2.8	
Embedded Controller Version	61.61	
BIOS Mode	UEFI	
BaseBoard Manufacturer	HP	
BaseBoard Product	81EB	
BaseBoard Version	61.61	
Platform Role	Mobile	
Secure Boot State	On	
PCR7 Configuration	Elevation Required to View	
Windows Directory	C:\WINDOWS	
System Directory	C:\WINDOWS\system32	
Boot Device	\Device\HarddiskVolume2	
Locale	United States	
Hardware Abstraction Layer	Version = "10.0.18362.387"	
User Name	SYSTEMBETA\System Administrator	
Time Zone	Eastern Daylight Time	
Installed Physical Memory (RAM)	8.00 GB	
Total Physical Memory	7.89 GB	
Available Physical Memory	2.93 GB	
Total Virtual Memory	9.21 GB	
Available Virtual Memory	3.26 GB	
Page File Space	1.31 GB	
Page File	C:\pagefile.sys	
Kernel DMA Protection	Off	
Virtualization-based security	Not enabled	
Device Encryption Support	Elevation Required to View	
Hyper-V - VM Monitor Mode Extensions	Yes	
Hyper-V - Second Level Address Translation Extensions	Yes	
Hyper-V - Virtualization Enabled in Firmware	No	
Hyper-V - Data Execution Protection	Yes	

[Sound]
Name	Realtek High Definition Audio	
Manufacturer	Realtek	
Status	OK	
PNP Device ID	HDAUDIO\FUNC_01&VEN_10EC&DEV_0282&SUBSYS_103C81EB&REV_1000\4&B91D09B&0&0001	

[Display]	
Name	Intel(R) HD Graphics 520	
PNP Device ID	PCI\VEN_8086&DEV_1916&SUBSYS_81EB103C&REV_07\3&11583659&0&10	
Adapter Type	Intel(R) HD Graphics Family, Intel Corporation compatible	
Adapter Description	Intel(R) HD Graphics 520	
Adapter RAM	1.00 GB (1,073,741,824 bytes)

**Comment by KirbyFan102** [comment]
Have you tried download the solar sensor patch and then patching the ROM with lunarips? I used to get a very similar error when I played the first time, but was able to run the game without problem after I patched it. Your mileage might vary, though.

**Comment by InfiniteParallel** [comment]
> Have you tried download the solar sensor patch and then patching the ROM with lunarips? I used to get a very similar error when I played the first time, but was able to run the game without problem after I patched it. Your mileage might vary, though.

Yes, I have tried this, Unfortunately, it made no difference in this type of error.
